FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Qemu_mps2/CMSIS/CMSDK_CM3.h

```diff
@@ -103,7 +103,7 @@ typedef enum IRQn
 /* --------  Configuration of the Cortex-M3 Processor and Core Peripherals  ------- */
 #define __CM3_REV                 0x0201      /* Core revision r2p1                              */
 #define __MPU_PRESENT             1           /* MPU present or not                              */
-#define __NVIC_PRIO_BITS          3           /* Number of Bits used for Priority Levels         */
+#define __NVIC_PRIO_BITS          8           /* Number of Bits used for Priority Levels         */
```

Comment:
Can we avoid changing this file, as it is a copy from CMSDK library

Reply:
do we need to set a note somewhere if someone updates the library, to make the necessary changes ? (possibly in the readme)